include/spdlog/spdlog-inl.h

```diff
@@ -89,4 +89,9 @@ SPDLOG_INLINE void apply_logger_env_levels(std::shared_ptr<logger> logger) {
     details::registry::instance().apply_logger_env_levels(std::move(logger));
 }
 
+SPDLOG_INLINE std::unique_ptr<spdlog::details::periodic_worker> &get_flusher()
+{
+    return details::registry::instance().get_flusher();
```

Comment:
This change is not needed, as the registry can be accessed

Reply:
It is not a must, I agree, but it allows to access directly from spdlog namespace (similar to spdlog::register_logger for example) which makes it neater IMHO.

Reply:
Better not add new api if not really necessary.

Reply:
There is a missing mutex lock in the `get_flusher()`

Add 
```c++
std::lock_guard<std::mutex> lock(flusher_mutex_);
```

Reply:
Got it. Fixed.